cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -16,6 +16,7 @@ const eventData = {
   sponsor_ids: [],
   chapter_id: 1,
 };
+
 describe('events dashboard', () => {
   beforeEach(() => {
     cy.exec('npm run db:seed');
```

Comment:
I can't comment on the next line, but could change it to log in as `admin@of.chapter.one`? We shouldn't need to be logged in as an owner to do this.

Reply:
the issue here is that there are 2 tests which broke:
`edit` and `delete` event
both of them trying to do operations on **first** event at homepage (which is not always event of chapter 1)

Reply:
may it can be good - to create different tests file, for `admin` user, which will have  `login('admin@of.chapter.one')` at its `beforeach` block

Reply:
I'd like to keep the tests together, if possible.  It's not perfect, but it is helpful to know which page is supposed to be under test.  We could split them further, but I'd rather keep the system simple (no more than one spec per type of page).

So, what to do here?  I think it's okay for now to just log in as the instance user where necessary (when operating on events that may not be in chapter one).  Ideally the tests should target specific chapters/events, but it's not priority number one!  If you could leave a comment about *why* it's necessary to change user, that should help people understand and know what to fix.

Reply:
will it be better if  create different `describe` section on the same file?

Reply:
Sure, that sounds good!